crates/biome_js_analyze/src/lint/style/use_consistent_type_definitions.rs

```diff
@@ -0,0 +1,247 @@
+                    if can_convert_interface_to_type(interface_decl) {
+                        return Some(DeclarationType::Interface(interface_decl.clone()));
+                    }
+                }
+                None
+            }
+            AnyJsDeclarationClause::TsTypeAliasDeclaration(type_alias) => {
+                if options.style == ConsistentTypeDefinition::Interface {
+                    // Check if type alias can be converted to interface
+                    if can_convert_type_to_interface(type_alias) {
+                        return Some(DeclarationType::TypeAlias(type_alias.clone()));
+                    }
+                }
+                None
+            }
+            _ => None,
+        }
+    }
+
+    fn diagnostic(ctx: &RuleContext<Self>, state: &Self::State) -> Option<RuleDiagnostic> {
+        let options = ctx.options();
+
+        let (range, current_style, preferred_style) = match state {
+            DeclarationType::Interface(interface_decl) => {
+                (interface_decl.range(), "interface", "type")
+            }
+            DeclarationType::TypeAlias(type_alias) => (type_alias.range(), "type", "interface"),
+        };
+
+        Some(RuleDiagnostic::new(
+            rule_category!(),
+            range,
+            markup! {
+                "Use "<Emphasis>{preferred_style}</Emphasis>" instead of "<Emphasis>{current_style}</Emphasis>"."
+            },
+        ))
+    }
+
+    fn action(ctx: &RuleContext<Self>, state: &Self::State) -> Option<JsRuleAction> {
+        let mut mutation = ctx.root().begin();
+
+        match state {
+            DeclarationType::Interface(interface_decl) => {
+                let type_alias = convert_interface_to_type_alias(interface_decl)?;
+                mutation.replace_node(
+                    AnyJsDeclarationClause::TsInterfaceDeclaration(interface_decl.clone()),
+                    AnyJsDeclarationClause::TsTypeAliasDeclaration(type_alias),
+                );
+
+                Some(JsRuleAction::new(
+                    ctx.metadata().action_category(ctx.category(), ctx.group()),
+                    ctx.metadata().applicability(),
+                    markup! { "Convert to "<Emphasis>"type"</Emphasis>" alias." }.to_owned(),
+                    mutation,
+                ))
+            }
+            DeclarationType::TypeAlias(type_alias) => {
+                let interface_decl = convert_type_alias_to_interface(type_alias)?;
+                mutation.replace_node(
+                    AnyJsDeclarationClause::TsTypeAliasDeclaration(type_alias.clone()),
+                    AnyJsDeclarationClause::TsInterfaceDeclaration(interface_decl),
+                );
+
+                Some(JsRuleAction::new(
+                    ctx.metadata().action_category(ctx.category(), ctx.group()),
+                    ctx.metadata().applicability(),
+                    markup! { "Convert to "<Emphasis>"interface"</Emphasis>"." }.to_owned(),
```

Comment:
```suggestion
                    markup! { "Use to "<Emphasis>"interface"</Emphasis>"." }.to_owned(),
```

Reply:
addressed - change to "Use" instead of "Use to"